src/bp/index.ts

```diff
@@ -108,6 +108,7 @@ try {
         process.IS_LICENSED = true
         process.ASSERT_LICENSED = () => {}
         process.BOTPRESS_VERSION = metadataContent.version
+        process.TELEMETRY_URL = process.env.TELEMETRY_URL || 'https://telemetry.botpress.com'
```

Comment:
is this the official endpoint ?

Reply:
no it is a template until the official one is deployed

Reply:
then add a TODO comment please

Reply:
Fixed in : e135fcf